platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/state/StateSigningTests.java

```diff
@@ -332,13 +327,13 @@ void signatureBecomesInvalidTest(final boolean evenWeighting) {
         assertEquals(expectedWeight, signedState.getSigningWeight());
 
         // Remove a node from the address book
-        final NodeId nodeRemovedFromAddressBook = nodes.get(0).getNodeId();
-        final long weightRemovedFromAddressBook = nodes.get(0).getWeight();
-        final AddressBook updatedAddressBook = signedState.getAddressBook().remove(nodeRemovedFromAddressBook);
-        signedState.getState().getWritablePlatformState().setAddressBook(updatedAddressBook);
+        final RosterEntry nodeToRemove = nodes.getFirst();
+        final Roster updatedRoster = signedState.getRoster();
+        assertTrue(RosterTestUtils.removeNode(updatedRoster, nodeToRemove.nodeId()));
```

Comment:
We no longer support this operation in the Roster world. It's immutable by definition. The only way to remove a node would be to create a new roster with an updated roster entries where that node is missing. This would yield a brand new instance of the Roster object. But we cannot and shouldn't do that in place in an already created Roster object.

Reply:
Like I mentioned above, this was all done to support what the original address book-based tests were doing. If the intent - which your comment makes it seem so - to not support such functionality for rosters, then I will remove that functionality and rework the tests.